Packs/BitwardenPasswordManager/Integrations/BitwardenPasswordManager/BitwardenPasswordManager.py

```diff
@@ -0,0 +1,236 @@
+    if events:
+        events = events[:max_fetch]
+        return events, CommandResults(readable_output=tableToMarkdown('Bitwarden Events', events),
+                                      raw_response=events)
+
+    return [], CommandResults(readable_output='No events found')
+
+
+def fetch_events(client: Client, max_fetch: int, start_date_str: str = DEFAULT_FIRST_FETCH) -> tuple[
+    List[Dict[str, Any]], Dict[str, Any]]:
+    last_run = demisto.getLastRun()
+    events, continuation_token = get_events(client, max_fetch, start_date_str, last_run)
+    unique_events = get_unique_events(events, last_run)
+    latest_events = filter_events(events=events, latest=True)
+    hash_latest_events = hash_events(latest_events)
+    if continuation_token:
+        demisto.debug(
+            f'Bitwarden - Fetched {len(unique_events)} which is the maximum or greater then the number of events.'
+            f' Will keep the fetching in the next fetch.')
+        created = start_date_str or last_run.get('last_fetch') or (
+            (get_current_time() - timedelta(minutes=1)).strftime(DATE_FORMAT))
+        new_last_run = {'continuationToken': continuation_token, 'last_fetch': created, 'nextTrigger': '0',
+                        'hashed_latest_events': hash_latest_events}
+    else:
+        # If there is no continuation token, the last fetch date will be the max end date of the fetched events.
+        new_last_fetch_date = max([dt for dt in (arg_to_datetime(event.get('date'), DATE_FORMAT)
+                                                 for event in unique_events) if dt is not None]).strftime(
+            DATE_FORMAT) if unique_events else get_current_time()
+        new_last_run = {'last_fetch': new_last_fetch_date, 'hashed_latest_events': hash_latest_events}
+        demisto.debug(f'Bitwarden - Fetched {len(unique_events)} events')
+
+    for event in unique_events:
+        event['_time'] = event.get('date')
+
+    return unique_events, new_last_run
+
+
+def get_events(client: Client, max_fetch: int, start_date_str: str, last_run: Dict[str, Any]) -> tuple[List[Dict[str, Any]], str]:
+    continuation_token = last_run.get('continuationToken', '')
+    events: List[dict] = []
+    has_next = True
+    while has_next:
+        has_next = False
+        if len(events) >= max_fetch:
+            break
+        start_date = last_run.get('last_fetch', '') if last_run.get('last_fetch', '') else start_date_str
+        response = client.get_events(start_date=start_date, continuation_token=continuation_token)
+        if continuation_token := response.get('continuationToken'):
+            has_next = True
+        events.extend(response.get('data'))
+    return events, continuation_token
+
+
+def get_unique_events(events: List[Dict[str, Any]], last_run: Dict[str, Any]) -> List[Dict[str, Any]]:
+    if hashed_latest_events := last_run.get('hashed_latest_events'):
+        recent_events = filter_events(events=events, latest=False)
```

Comment:
continuing the previous comment, for me it sounds like now you need the oldest events, not the recent ones